src/components/color_picker/color_picker_entry.xml

```diff
@@ -0,0 +1,20 @@
+<templates>
+  <t t-name="o-spreadsheet-ColorPickerEntry" owl="1">
+    <div class="o-dropdown o-color-picker-entry">
+      <div
+        class="o-tool o-with-color"
```

Comment:
No guarantees that these class are defined where we use the colorPicker. Move the class definition inside the component if its shared between all the colorPickers, or make them props